ingeniamotion/utils/monitoring.py

```diff
@@ -0,0 +1,24 @@
+from typing import Union
+
+from ingenialink.ethernet.register import EthernetRegister
+from ingenialink.canopen.register import CanopenRegister
+
+
+def map_register_address(register: Union[EthernetRegister, CanopenRegister]) -> int:
```

Comment:
This function could be a static method of Monitoring or Capture. Right? Do we need a utils folder for this?

Reply:
This method is also used in disturbance, I don't like the idea of using a static method from Monitoring there. I could not think of a better place to put this function besides in a utils module.

Reply:
We have a circular import error when moving the `map_register_address `function to Capture. I think I'll revert back to using a utils module.